docs/learn/software_overview.md

```diff
@@ -1,53 +1,54 @@
-and tools, to give you a better understanding of some of the terminology, which gives you the
-language to find help. Also, we want you to have an overview of the area, so you can understand all
-of the parts, and how they work together.
+Starting out in the [CNC](https://en.wikipedia.org/wiki/Numerical_control) world can be very intimidating, even if you already know about 3D printing (which is a CNC machine, technically).  However, when broken down into manageable pieces it's easy to digest.  In the world of CNC, workflow and software are intrinsically linked. 
+
+Here, we will describe the different steps in the CNC milling workflow process along with their associated software components/tools.  The goal is to provide you a better understanding of some of the steps and terminology, which in turn gives you the background to better seek help and explore possibilities.
 
 ## Overview
 
-This is the 1000ft (304m) overview.
+This is the 1000ft (304m) overview (we will explore each in more detail within the sections that follow).
+
+There are 4 components to the workflow and associated software as follows:
 
-There are 4 parts: CAD, CAM, Sender, Controller
+| |Step|Software|
+|-|----|--------|
+|1|Design|CAD (Computer Assisted Design) and/or VGE (Vector Graphics Editor)|
+|2|Toolpathing|CAM (Computer Assisted Machining)|
+|3|G-Code Sending|Sender|
+|4|Milling|Controller/Firmware|
 
 TODO Insert a diagram showing the workflow.
 
-[CAD: Computer Assisted Design](#cad-computer-assisted-design)
-:   Before you do anything, you have to take your idea and get it into the computer. This can take
-    many forms, but the output is either a 2D file (like a .DXF or .SVG) or sometimes (but not
-    often) a 3D file (like a .STL).
+[Design/CAD/Vector Graphics Editor](#design-cad-vge)
+:   For the most part, you are either going to use your CNC machine to create a part (e.g. a wooden or metal gear) or for something         artistic (e.g. a sign or carving). If the circumstance is that you are needing to create the design from scratch you will likely be       using CAD or Vector Graphics Editor software to do so.  It is worth noting that this is NOT a certainty.  Many people may already       have these designs in hand provided from some other source.  Regardless, the output of this step (and input to the next) is             typically either a 2D file (like a .DXF or .SVG) or sometimes (but not often) a 3D file (like a .STL).
 
-[CAM: Computer Assisted Machining](#cam-computer-assisted-machining)
-:   This is the step where you take your design, and describe which lines to trace, which areas to
-    cut out, and which parts to leave. CAM also is where you will define what kinds of bits you
-    have, and how fast and deep you want to move them through the material. The output of this step
-    is g-code (in a .gcode or .nc file).
+[Toolpathing/CAM](#toolpathing-cam)
```

Comment:
Is "toolpathing" something people will recognize? I understand people won't know what CAM is, but this is the reason for this document, right? I think adding a new term may confuse the issue.

Reply:
That is a good question.  Let me explain be personal experience and then you be the judge.  I am not saying this is right, just how I looked at it.

When I first hear the term Computer Assisted Machining I thought "ya, duh, that is what a CNC does".  I think I also heard it in my head Computer Assisted Milling.  In other words, the software that physically ran the CNC (like maybe the role of the g-code sender).  Then, when I was trying to learn, in many video's folks refer to "toolpath".  The video basically say something like "...okay open up your CAM software so we can define the toolpath for the circle".

I was then like "oh, I get it".  CAM is actually defining the path of your tool.  I wish someone told me that from the start.

If memory serves, I found others using that term quite a bit.  If you search youtube for "toolpath" I think its common.

Again, just my reasoning, not saying it correct.

Reply:
"Toolpaths" or "Tool Paths" are definitely a thing. But in the same way that CNC is a term, that just needs to be defined, I think CAM is a term that just needs to be defined. Maybe the trouble is even in explaining the definition. I haven't seen "Toolpathing" before to describe CAM, and I have only seen CAM and "Slicer" used to define this step. I think one of the major takeaways people will have when they read this doc, and aren't familiar with CNC is what CAM is, and where it is in the process. Even Estlcam gets confusing because it is also a firmware and G-Code sender.

I will also work on getting an image that shows the flow. It will be a simple thing with a few boxes and I think it will make this all a little easier/lower stress.